packages/website/docs/creating-a-renderer.md

```diff
@@ -3,7 +3,13 @@ id: creating-a-renderer
 title: Creating a Renderer
 ---
 
-Learn how to build an autocomplete UI using React.
+Learn how to build an autocomplete UI from the ground up, focusing React.
```

Comment:
The fact that it uses React is an afterthought here, it could be with anything else, but that's not the point.

I think we can leave it out of the first lines to ensure people don't associate the need for a custom renderer with usage in React.

```suggestion
Learn how to build a full autocomplete UI from the ground up.
```